scipy-1.0/paper.tex

```diff
@@ -28,7 +28,19 @@
 \keywords{Scientific computing, Python, Mathematics}
 
 \begin{abstract}
-TODO. Abstract must be under 200 words and not include subheadings or citations.
+SciPy is an open source scientific computing library for the Python programming language.
+SciPy 1.0 was released in late 2017, about 16 years after the original
+version 0.1 release. SciPy
+has become a \emph{de facto} standard in the community, with more than 600
+unique code contributors and millions of downloads per year, despite
+the absence of an official project publication, until now. The library
+includes functionality spanning clustering, Fourier transforms,
+integration, interpolation, file I/O, linear algebra, image processing,
+orthogonal distance regression, minimization algorithms, signal processing,
+sparse matrix handling, computational geometry, and statistics. In this
+work, we highlight some recent technical developments and provide
```

Comment:
I typically refrain from making suggestions I can't back up with objective rationale... but for whatever reason, I feel strongly enough to suggest reversing the order of "overview of the capabilities and development practices..." and "highlight some recent technical developments". Perhaps it's for the same reason that we begin the paper with overviews and end with specifics. But if nobody else suggests the same, please ignore me : )